pages/en/contrib/LTS.md

```diff
@@ -23,7 +23,7 @@ Below is the LTS schedule on the LoopBack versions:
 
 Framework | Status | Published | Active LTS Start | Maintenance LTS Start | EOL | Runtime | GA | EOL
 -- | -- | -- | -- | -- | -- | -- | -- | --
-LoopBack 4 | Current | Oct 2018 | -- | -- | Apr 2021 _(minimum)_| Node 10 | Oct 2018 | Apr 2021
+LoopBack 4 | Current | Oct 2018 | -- | -- | Apr 2021 _(minimum)_| Node&nbsp;10.16 | Oct 2018 | Apr 2021
```

Comment:
@andrey-abramow @achrinza, IIUC, Node.js LTS schedule goes by the major versions, see https://nodejs.org/en/about/releases/, so I'm not sure if it's necessary to make the change in here.